### PR TITLE
(chore) fix the url in #![doc(html_root_url)]

### PR DESCRIPTION
**Pull request by ubnt-intrepid**
See the Rust API guideline ([link][1]) for details.

[1]: https://rust-lang-nursery.github.io/api-guidelines/documentation.html#crate-sets-html_root_url-attribute-c-html-root

### CODE REVIEW

**Comment by sunng87** [comment]
Thank you for this patch. For adding version number to the line, can you please use cargo-release's replacement hook? You can see release.toml for detail.

**Comment by ubnt-intrepid** [comment]
@sunng87 added entries to replacement hook.

File: src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#![doc(html_root_url = "https://docs.rs/crate/handlebars/")]
+#![doc(html_root_url = "https://docs.rs/handlebars/CrateVersion")]
```
Comment (sunng87) [inline src/lib.rs]: There is an issue if you hard-coded `CrateVersion` here that once it's replaced in a release. There will be no such `CrateVersion` to replace in future release. You can use regex to capture the version number here.

There is an example in cargo-release's release.toml as example. Replying from mobile, sorry for unable to link it.

Comment (ubnt-intrepid) [reply src/lib.rs]: I've fixed in e6de93a.

**Pull request merged**